packages/core/src/access.ts

```diff
@@ -9,7 +10,7 @@ export function denyAll() {
   return false;
 }
 
-export function unfiltered() {
+export function unfiltered<ListTypeInfo extends BaseListTypeInfo>(): MaybePromise<boolean | ListTypeInfo['inputs']['where']> {
```

Comment:
The type ends up inferring as `=> boolean`, which isn't helpful when you want the type to help infer other function types.